index.js

```diff
@@ -1,3 +1,4 @@
 export { parse } from "./lib/webidl2.js";
 export { write } from "./lib/writer.js";
 export { validate } from "./lib/validator.js";
+export {Err} from "./lib/tokeniser.js";
```

Comment:
This is good, but I would prefer the original name `WebIDLParseError`.

Reply:
Make sure we get spaces between braces!

Reply:
Okay I am adding spaces now it is also passing all the tests, so I think it can be merged after that change :)